eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreWallet.scala

```diff
@@ -49,6 +51,13 @@ class BitcoinCoreWallet(rpcClient: BitcoinJsonRPCClient)(implicit ec: ExecutionC
     rpcClient.invoke("signrawtransaction", hex).map(json => {
       val JString(hex) = json \ "hex"
       val JBool(complete) = json \ "complete"
+      if (!complete) {
+        val message = json \ "errors" match {
+          case value: JValue => Serialization.write(value)(DefaultFormats)
```

Comment:
We should look in the code of bitcoin core what is normally returned in the `errors` field for `signrawtransaction` try to parse it in a best effort basis here, and return "signrawtransaction failed" if we failed.